<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车票管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="runsInformation">
     <attribute name="title">
      <string>班次信息</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>531</width>
        <height>201</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>529</width>
         <height>199</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="runsInformationTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>531</width>
          <height>201</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>班次</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>发车时间</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起点站</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>终点站</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>行车时间</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>额定载量</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>已售票</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="buyButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>购票</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refundButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退票</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="routeSearch">
     <attribute name="title">
      <string>路线查询</string>
     </attribute>
     <widget class="QRadioButton" name="byRunsRadioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>按班次查询</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="byStationRadioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>按终点站查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="runsNumberLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>班次号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="runsNumberLineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>404</x>
        <y>100</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="startStationLineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="endStationLineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="startStationLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>起点站：</string>
      </property>
     </widget>
     <widget class="QLabel" name="endStationLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>终点站：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="searchResult">
     <attribute name="title">
      <string>查询结果</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>531</width>
        <height>201</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>529</width>
         <height>199</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="searchRunsInformationTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>531</width>
          <height>201</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>班次</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>发车时间</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起点站</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>终点站</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>行车时间</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>额定载量</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>已售票</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="searchRefundButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退票</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchBuyButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>购票</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>车票管理系统</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actionAddRuns"/>
    <addaction name="actionDeleteRuns"/>
    <addaction name="actionAlter"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAddRuns">
   <property name="text">
    <string>增加班次</string>
   </property>
  </action>
  <action name="actionDeleteRuns">
   <property name="text">
    <string>删除班次</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>从文件读取班次</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存班次到文件</string>
   </property>
  </action>
  <action name="actionAlter">
   <property name="text">
    <string>修改班次</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>getInformation(QString,QString,QString,QString,QString,QString,QString)</slot>
 </slots>
</ui>
